Xamarin.Forms.Core/StackLayout.cs

```diff
@@ -1,6 +1,10 @@
 using System;
 using System.ComponentModel;
+using Xamarin.Forms;
 using Xamarin.Forms.Internals;
+using Xamarin.Forms.StyleSheets;
+
+[assembly: StyleProperty("-xf-stacklayout-spacing", typeof(StackLayout), nameof(StackLayout.SpacingProperty))]
```

Comment:
this one is the first in which it make sense to have `vendor-type-property`

Reply:
Huh. Wonder why wasn't it included in the original spec and why the original spec has so many other ones specified... This one was requested in a comment from a customer.

Reply:
what @samhouts  said

Reply:
`-xf-spacing`